src/Access/AuthenticationData.h

```diff
@@ -83,6 +87,7 @@ class AuthenticationData
     String kerberos_realm;
     boost::container::flat_set<String> ssl_certificate_common_names;
     String salt;
+    std::vector<ssh::SSHKey> ssh_keys;
```

Comment:
Should be set and in opreator== compared properly, it is already done in https://github.com/ClickHouse/ClickHouse/pull/48902